<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicUserDialog</class>
 <widget class="QWidget" name="MusicUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>381</width>
     <height>256</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>用户登录</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>7</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="one">
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>35</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>帐号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwdLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>79</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="userComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>37</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="forgotPwdButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>79</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>忘记密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>35</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>注册帐号</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="rememberPwd">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>137</y>
       <width>90</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="automaticLogon">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>137</y>
       <width>90</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>自动登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="useTencentLogin">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>180</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/usermanager/lb_qq)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="useRenrenLogin">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>180</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/usermanager/lb_renren)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="useWechatLogin">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>180</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/usermanager/lb_weixing)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="useSinaLogin">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/usermanager/lb_xinlang)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="two">
    <widget class="QLabel" name="usernameRegisterLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>40</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>注册帐号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="mailLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>100</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>邮箱地址：</string>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="registerMailLine">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>98</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="registerUserLine">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>35</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit::hover{ border: 2px solid #AAAAFF;}</string>
     </property>
    </widget>
    <widget class="QLabel" name="registerUserLineR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>4-20个字母、数字和符号组成</string>
     </property>
    </widget>
    <widget class="QLabel" name="registerMailLineR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>133</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请填写正确的邮箱地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="registerPwdLineR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>195</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>6-16个字母、数字和符号组成</string>
     </property>
    </widget>
    <widget class="QLabel" name="registerPwdCLineR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>255</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>6-16个字母、数字和符号组成</string>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="registerPwdLine">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>160</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="registerPwdCLine">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>220</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="passwdLable">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>162</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>设置密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwdCLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>223</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="agreementCheckBox">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>300</y>
       <width>125</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>我已阅读并接受</string>
     </property>
    </widget>
    <widget class="QPushButton" name="agreementButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>网站协议</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRight">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelRigh_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelRigh_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>162</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelRigh_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>222</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="three">
    <widget class="QLabel" name="userLineLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>38</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>原帐号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="mailLineLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>87</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>原邮箱：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mailLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>85</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="userLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>35</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="pwdLineLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>136</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码：</string>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="pwdLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>135</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="pwdLineEditR">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>170</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>6-16个字母、数字和符号组成</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRigh_5">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>140</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="MusicCodeAreaWidget" name="verificationCode">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="verificationCodeEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="verificationCodeLabel">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>214</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>输入验证码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicUserLineEdit</class>
   <extends>QLineEdit</extends>
   <header>musicuserlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicCodeAreaWidget</class>
   <extends>QLabel</extends>
   <header>musiccodeareawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
